### ISSUE TITLE
Update blurb next to video

### ISSUE THREAD

**Issue by humancompanion** [USAJOBS/Help#134]
I have the link from Sarah, but we also need to replace the blurb that goes next to the video. @skpark19 - Can you whip up a paragraph about the release, run it by @dalenard21, and drop it in this issue? Thanks.

**Comment by skpark19** [comment]
USAJOBS has released a streamlined account designed to more clearly communicate application statuses, saved jobs, and saved searches. Resumes and other documents are now collocated in the Documents section of Account, and tips and completion indicators have been added to better prepare applicants to apply for Federal jobs.

**Comment by dalenard21** [comment]
@skpark1 - thanks.

I know this is approved and I don't want to add any more delays, but for an upcoming release, I'd like to slightly edit the content to put it into the active voice.  Active voice is more concise  -- it uses fewer words and tends to be a bit friendlier. Plainlanguage.gov also recommends using the active voice.

For example: 

Instead of: "...and tips and completion indicators have been added to better prepare application to apply for Federals jobs"

Rewrite to: "We've added tips and completion indicators to..."

thanks.
